### DISCUSSION TITLE
Screen glitches with BTT SKR E3V3 board

### DISCUSSION THREAD

**Discussion by PunishedLemur** [mriscoc/Ender3V2S1#432] [General]
I have the following setup, Ender3V2, BTT SKR E3V3.
My Ender 3 v2 came with the dacai display.  I'm running the latest version of the professional fw from a few days ago, but I have been running it for about a month now and have always had issues with display corruption.
At first, I was using just the firmware without having updated the display, but since then I updated the display successfully using the private folder with the stock icon set.
Things did improve some after the display FW update via the private folder, the corruption is not as bad or as frequent.
I've attached two photos, first is what I was getting before display fw update, the second is what I am seeing after, and running the current version from the 4th.  Largely, numbers disappear now, or appear very large and superimposed over the area.

Some things I have tried so far without any change. 
Turning 115k baud off.
Rerouting to attempt to account for interference, and even remaking a second custom display cable for the SKR mini.
Disabling all the toolbar extras.

![IMG_20221005_222024204](https://user-images.githubusercontent.com/115188519/194726583-722f7d17-ce33-4bf1-8901-df15db675663.jpg)
![IMG_20221006_141918960](https://user-images.githubusercontent.com/115188519/194726588-c0089c51-1081-4aa4-85df-efe1f9ac733f.jpg)

**Comment by mriscoc** [comment]
Try re-routing the screen cable and with a ferrite ring.

**Reply by PunishedLemur** [reply]
I took the one off of the old cable that came with the stock printer, and put it on, but no luck.  Still getting artifacts.
Should the ferrite be closer to the board or the display?  I put it close to the board.

**Reply by PunishedLemur** [reply]
I went further in wrapping the whole cable in foil, and re-routing it so it passes the absolute minimum of cables before exiting the electronics enclosure, Still I see corruption in the number sizes.

Can I just comment out the pro ui and use the marlin ui in the firmware as a test?  Will I lose a lot of needed functionality or still be able to use all the main features?  Other than the obvious ones like the toolbar and other ui specific features.

**Reply by mriscoc** [reply]
Removing the blinking leveling indicator can also helps, comment out these lines:
https://github.com/mriscoc/Ender3V2S1/blob/2f8c9389b86d8468b386bb5b2f28c00de7526816/Marlin/src/lcd/e3v2/proui/dwin.cpp#L635-L638

**Reply by PunishedLemur** [reply]
Edited (Disregard, the problem was caused because I also commented out preview and lock screen under proui features in configuration H, I reverted that and am trying a new build with the blinking indicator disabled as suggested.)

Fails to compile after commenting those out.
![Capture2](https://user-images.githubusercontent.com/115188519/195739032-85e2966b-dbef-4633-b855-e94a5b8e7ba8.PNG)
![Capture](https://user-images.githubusercontent.com/115188519/195739042-d9db97f3-1a79-45f7-be85-dc9c07d2d0a8.PNG)

**Comment by mriscoc** [comment]
There are general problems with the display and the SKR Mini board, check https://github.com/mriscoc/Ender3V2S1/issues/445

**Reply by PunishedLemur** [reply]
Hmm, yeah this is the same issue I have.  It's still there after changing those lines.

At this point it's better than it was and doesn't do it as often at least.  I ordered another ender v2 display from ali express in the hopes I get a dwin rather than dacai model.

**Reply by H0Z0** [reply]
Unfortunately, DWIN is not better. There are also glitch with DWIN. Also with 115k and ferrite ring. There are fewer errors at 115k, but before that there were much more with DWIN too. You don't really get DWIN anyway if you order a new one.
